### DISCUSSION TITLE
getting: Uncaught DOMException: Failed to construct 'CustomElement'

### DISCUSSION THREAD

**Discussion by mathisloge** [vmware/clarity#6609] [Q&A] [answered]
So I just wanted to test cds and after setting everything up, I'm getting 
`Uncaught DOMException: Failed to construct 'CustomElement': The result must not have attributes`
for the element: `cds-navigation`

I've made a repro here https://github.com/mathisloge/cds-nav-repro 

I'm on latest ms edge chromium. 

Do I have done anything not correct?

**Comment by hippee-lee** [comment, marked as answer]
Hi @mathisloge - I'm not seeing where the navigation component is registered in the repo linked to. Am I missing where you register it as described [here](https://clarity.design/storybook/core/?path=/story/components-navigation--page)? 
```javascript
import '@cds/core/navigation/register.js';
```

**Reply by mathisloge** [reply]
Shouldn't the CdsModule take care of it? At least the docs and examples told me so 
However, I can also use other components. And I think with angular 11 or something, it worked.

**Reply by mathisloge** [reply]
https://stackblitz.com/edit/clarity-dark-theme-v5-1t2cf2

**Reply by hippee-lee** [reply]
Yeah, you are correct. I'll investigate it with a new repo on my local computer and let you know. Stackblitz is throwing weird errors with the latest sources but its hard to debug build stuff there.

**Reply by hippee-lee** [reply]
It looks like there is already a [fix](https://github.com/vmware/clarity/pull/6593) for this issue that will go out with the next release.

**Reply by mathisloge** [reply]
Oh yeah. Don't know how I missed that. Sorry!

**Reply by hippee-lee** [reply]
No worries, I did to until a teammate pointed it out 😳
